docs/Guides/QuickStart.md

```diff
@@ -28,3 +28,46 @@ If you like learning through watching, check out our getting started Video:
 <video height="1080" controls>
   <source src="https://github.com/ZOSOpenTools/collateral/raw/main/ZOSOpenToolsIntroV2-cropped.mp4" type="video/mp4">
 </video>
+
+
+## Upgrading the zopen packages
+
+Use this process to upgrade existing packages
+
+```
+$ zopen upgrade
+```
+
+
+## Upgrading the zopen package manager
+
+Use this process to upgrade the zopen package manager (meta).
+By default, `zopen upgrade` updates the individual packages.
+In order to update the package manager (meta), it needs
```

Comment:
i don't believe the package manager has to be done separately so i think you can safely delete this sentence. @DevonianTeuchter please comment if otherwise

Reply:
Yes, can be done at the same time - only reason for it being called out was when upgrading from a pinned (pre-fork-merge) version where .pinned was set for meta.

Reply:
Ok. I will try to capture the situation.  Please review the next update.